test-projects/demo/dashboards/welcome.visivo.yml

```diff
@@ -0,0 +1,82 @@
+models:
+  - name: Waterfall Model
```

Comment:
@jpjsonic I would like to replace this with a CSV.  You know off the top of your head what does this data look like?

Reply:
Yup, I can push up a quick PR that turns this into a csv 👍